src/main/java/uk/gov/justice/digital/hmpps/whereabouts/security/VerifyAgencyAccess.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.whereabouts.security;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface VerifyAgencyAccess {
```

Comment:
Nothing reads this - will it or can we get rid of it?

Reply:
It's actually used here: https://github.com/ministryofjustice/whereabouts-api/pull/116/files#diff-da6a8798b7f4b4480af332901fa44352R26

Reply:
but nothing's reading it is it? 

Guess this was copied over from elite2 impl, Are you going to reimplement some aspect-y thing that checks against elite2 in a follow up or are you removing the check?

Reply:
Ah thanks, I see your point.  As we call elite2 first to get the location groups from file, I can catch the error that returns if the user is not authorised to the agency.

Reply:
Yes can be removed